hack/reduce-snapshot.sh

```diff
@@ -20,10 +20,10 @@
 # It requires that the following environment variables be defined:
 #
 # - SINGLE_COMPONENT: true if single component mode is enabled.
-# - SNAPSHOT: String representation of Snapshot
+# - SNAPSHOT: Path to Snapshot json file
```

Comment:
Looks like it could be either a string representation of snapshot or a path to snapshot json file